autolens/lens/ray_tracing.py

```diff
@@ -480,8 +468,11 @@ def image_plane_multiple_image_positions(self, grid, source_plane_coordinate):
             mask_2d=trough_mask,
         )
 
-        return grids.Coordinates.from_pixels_and_mask(
-            pixels=[multiple_image_pixels], mask=trough_mask
+        return list(
```

Comment:
You only need to cast to list if you're going to use indexing. For iteration leaving the object as a generator can be more efficient.